mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -117,16 +117,33 @@ view: +subscriptions {
     sql: 1;;
   }
 
-  dimension: months_retained {
+  dimension: months_active {
     type: number
+    # month is timezone sensitive, so use localized datetime to calculate months
     sql: mozfun.norm.subscription_months_renewed(
-      -- month is timezone sensitive, so use localized datetime to calculate monthly retention
       DATETIME(${subscription_start_raw}, ${plan_interval_timezone}),
       DATETIME(${end_raw}, ${plan_interval_timezone}),
       ${billing_grace_period}
     );;
   }
 
+  dimension: current_timestamp_limit {
+    hidden: yes
+    description: "Maximum possible timestamp for which data is available. Assumes ETL is complete up to CURRENT_DATE."
+    sql: DATETIME_SUB(TIMESTAMP(CURRENT_DATE), INTERVAL 1 MICROSECOND);;
```

Comment:
I'm going to add this as a field and `months_active` to the bigquery-etl query so that it is correct between UTC midnight and whenever ETL completes, but until then this uses `CURRENT_DATE`

also, note that bigquery `TIMESTAMP`s have microsecond resolution, so this is the maximum timestamp for "yesterday"